.github/workflows/release.yml

```
@@ -13,23 +13,21 @@ on:
       - v[0-9]+.[0-9]+.[0-9]+
 
 env:
+  ARCH: x86_64
+  ZIG_VERSION: 0.13.0
   BIN_DIR: ${{ github.workspace }}/bin
-  MINIMUM_CMAKE_VERSION: '3.28'
 
 # Build on the oldest supported images, so we have broader compatibility
 # Build with gcc-10 to prevent triggering #14150 (default is still gcc-9 on 20.04)
 jobs:
   linux:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
```

```suggestion
    runs-on: ubuntu-latest
```
(24.04 is still in beta)

---

24.04 comes with cmake 3.28 (our min version). 23.10 comes with cmake 2.27

---

Minimum version is 3.13 tho.

---

ah ok. The MINIMUM_CMAKE_VERSION var lead me to believe it was 3.28